schemas/documents/registry/profilePage.js

```diff
@@ -0,0 +1,15 @@
+export default {
+  name: 'profilePage',
+  type: 'document',
+  title: 'Profile Page',
+  fields: [
+    {
+      title: 'Banner Card',
+      name: 'bannerCard',
+      type: 'bannerCard',
+      description:
+        'This card gets displayed the first time a user accesses his/her own profile page',
+      validation: Rule => Rule.required(),
```

Comment:
Do we need these pages to be required? I think it might actually be nice if at hte Sanity level they were optional, and therefor the cards would only get rendered if a corresponding banner was defined in Sanity.

Reply:
sure we can make them optional, on the front-end side, the code was already set up to only render the banner if there's some content in sanity